db/migrations/009_create_excused_sessions.rb

```diff
@@ -0,0 +1,11 @@
+Sequel.migration do
+    change do
+        create_table(:excused_sessions) do
+            DateTime :date, null: false
```

Comment:
Same here; better to have an integer primary key; then you can get rid of the hack below.